### PR TITLE
Tensor-src-tizensensor: fixes (style & build)

### PR DESCRIPTION
**Pull request by myungjoo**
Apply gst-indent to the new element, "tensor_src_tizensensor".
    
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

Changed in v2:
- Fixed meson.build
- Fixed all the build erros and warnings of tizensensor

### CODE REVIEW

File: ext/nnstreamer/tensor_source/tensor_src_tizensensor.c [hunk 1]
```diff
@@ -491,21 +1044,206 @@ static GstFlowReturn
...
+  if (FALSE == self->configured) {
+    GST_ERROR_OBJECT (self,
+        "gst_tensor_src_tizensensor_fill() cannot proceed if it is not configured.");
+    retval = GST_FLOW_ERROR;
+    goto exit;
+  }
+
+  if (size != gst_tensor_info_get_size (self->src_spec)) {
+    GST_ERROR_OBJECT (self,
+        "gst_tensor_src_tizensensor_fill() requires size value (%u) to be matched with the configurations of sensors (%lu)",
```
Comment (myungjoo) [inline ext/nnstreamer/tensor_source/tensor_src_tizensensor.c]: %lu is not portable for 32/64b. will change it soon.

File: ext/nnstreamer/tensor_source/tensor_src_tizensensor.c [hunk 2]
```diff
@@ -907,27 +902,25 @@ gst_tensor_src_tizensensor_event (GstBaseSrc * src, GstEvent * event)
...
 
   if (FALSE == self->configured || SENSOR_ALL == self->type || NULL == spec) {
-    tensor = g_strdup_printf("other/tensor; other/tensors, num_tensors=1");
+    tensor = g_strdup_printf ("other/tensor; other/tensors, num_tensors=1");
   } else {
-    typestr = gst_tensor_get_type_string (spec->tinfo.type);
+    const gchar *typestr = gst_tensor_get_type_string (spec->tinfo.type);
     g_assert (typestr && typestr[0]);
-    tensor = g_strdup_printf("other/tensor, dimension=%u:%u:%u:%u, type=%s ; "
+    tensor = g_strdup_printf ("other/tensor, dimension=%u:%u:%u:%u, type=%s ; "
```
Comment (jaeyun-jung) [inline ext/nnstreamer/tensor_source/tensor_src_tizensensor.c]: TODO: consider to use gst_tensor_get_dimension_string()

File: ext/nnstreamer/tensor_source/tensor_src_tizensensor.c [hunk 3]
```diff
@@ -247,97 +248,101 @@ static GHashTable *tizensensors = NULL;
...
-                 .dimension={1, 1, 1, 1} }},
-    { .type=SENSOR_HUMAN_SLEEP_DETECTOR, .value_count=1, /** @todo check! */
-      .tinfo = { .name="values", type=_NNS_FLOAT32,
-                 .dimension={1, 1, 1, 1} }},
-    { .type=SENSOR_HUMAN_STRESS_MONITOR, .value_count=1, /** @todo check! */
-      .tinfo = { .name="values", type=_NNS_FLOAT32,
-                 .dimension={1, 1, 1, 1} }},
-    { .type=SENSOR_LAST, .value_count=0, .tinfo = {0, }},
+  {.type = SENSOR_ACCELEROMETER,.value_count = 3,
+        .tinfo = {.name = valstr,.type = _NNS_FLOAT32,
```
Comment (jaeyun-jung) [inline ext/nnstreamer/tensor_source/tensor_src_tizensensor.c]: Question: is it necessary to set name 'values' in GstTensorInfo? (NULL if possible)

Comment (myungjoo) [reply ext/nnstreamer/tensor_source/tensor_src_tizensensor.c]: No. it's not. So removed :)

File: ext/nnstreamer/tensor_source/tensor_src_tizensensor.c (hunk 2 above)
Comment (myungjoo) [reply ext/nnstreamer/tensor_source/tensor_src_tizensensor.c]: Fixed.

File: ext/nnstreamer/tensor_source/tensor_src_tizensensor.c [hunk 4]
```diff
@@ -1091,30 +1079,30 @@ gst_tensor_src_tizensensor_create (GstBaseSrc * src, guint64 offset,
...
 {
-    switch (spec->type) {
-      case _NNS_FLOAT32:
-        mamcpy (map->data, values, sizeof(float) * count);
-        break;
+  switch (spec->type) {
+    case _NNS_FLOAT32:
+      memcpy (map->data, values, sizeof (float) * count);
+      break;
       case_cast_loop (values, count, map->data, int64_t, _NNS_INT64);
```
Comment (dongju-chae) [inline ext/nnstreamer/tensor_source/tensor_src_tizensensor.c]: Need to shift `case_cast_loop()` to the left?

Comment (myungjoo) [reply ext/nnstreamer/tensor_source/tensor_src_tizensensor.c]: gst-indent won't recognize it and emit error if I do that :(

Comment (dongju-chae) [reply ext/nnstreamer/tensor_source/tensor_src_tizensensor.c]: Oh... i see.

**Pull request merged**